fix(doc): attempt to enhance the License section

jjargot opened the pull request:
Change the content of the *License* section because 100% of the customers are failing to start the application server after a migration.

Versions: 7.3, 7.4, 7.5, 7.6

jjargot committed:
fix(doc): attempt to enhance the License section

Change the content of the *License* section because 100% of the customers are failing to start the application server after a migration.

Versions: 7.3, 7.4, 7.5, 7.6

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/signed)](https://cla-assistant.io/bonitasoft/bonita-doc?pullRequest=428) <br/>All committers have signed the CLA.

jjargot committed:
fix(doc): changes content of license after viewing the markdown rendering

changes content of license after viewing the markdown rendering

Versions: 7.3, 7.4, 7.5,

ugaston reviewed (approved):
Looks fine to me.

Thanks Jérôme!

jjargot commented:
Thank you to have rewrote my first version.

tbouffard reviewed (approved):
👍 for me too, you can merge the PR

educhastenier commented:
retest this please

educhastenier committed:
improve layout / formatting

educhastenier commented:
test this please

educhastenier commented:
test this please
